src/snovault/local_storage.py

```diff
@@ -0,0 +1,133 @@
+import binascii
+
+from os import urandom
+from datetime import datetime
+from pytz import timezone
+
+from redis import StrictRedis
+from pyramid.view import view_config
+
+from snovault.interfaces import (
+    LOCAL_STORAGE,
+    LOCAL_STORAGE_EVENT_TAGS,
+    LOCAL_STORAGE_STATE,
+)
+
+
+def includeme(config):
+    config.scan(__name__)
+    config.add_route('local_storage', '/local_storage')
+    config.registry[LOCAL_STORAGE] = LocalStoreClient(
+        db=config.registry.settings['local_storage_db'],
```

Comment:
this is correctly set as a registry setting, however, I wonder if there's any hidden pyramid magic that can give you the application name (main, indexer, regionindexer, etc...) and then have a hash map making the translation to the DB index number. If there isn't, maybe we could rename the variable to `local_storage_db_redis_index` or something like that, people might get lost when they see that for the first time. Or add a comment in the .ini files

Reply:
yea, the naming could be better.  There are a few others I might fix along with this one.

The only way I know to get the name of the section in the config is to add a var called name to the section in the config :)    We sorta do this in the ini files for encoded with the set indexer = true, set visindexer = true...at one point I named them explicitly.  Must have been in a feature that didn't get added.

Reply:
oh, and yea.  I think the db indexes should be global to all processes.  How would we get indexer data in the app if the app didn't know the indexer db index.